src/commands.ts

```diff
@@ -105,7 +108,7 @@ function closeCommand(ctx: Context) {
   if (cache.config.categories) {
     cache.config.categories.forEach((element: any, index: number) => {
       // No subgroup
-      if (cache.config.categories[index].subgroups.length > 0) {
+      if (cache.config.categories[index].subgroups == undefined) {
```

Comment:
That's right. However, if no `subgroups` are specified, an error occurs. If it has a length of 0, `forEach` will be ignored. `cache.config.categories[index].subgroups == undefined && cache.config.categories[index].subgroups.length > 0` perhaps it would be more correct.